analysis/R/decode.R

```diff
@@ -211,42 +218,27 @@ ComputePrivacyGuarantees <- function(params, alpha, N) {
 }
 
 FitDistribution <- function(estimates_stds, map, quiet = FALSE) {
```

Comment:
Just a small nitpick. The indentation for the FitDistribution function seems off compared to the rest of the file.